week5/src/list.js

```diff
@@ -0,0 +1,8 @@
+function List({item, deleteit}){
```

Comment:
nit: naming: 1. Why `deleteit`? `simply `delete` is Ok here 2. JS using something called camel case for naming variables, you should name it `deleteIt` instead of `deleteit` if you wanted to use this variable name

Reply:
I didn't use delete cause just "delete" wouldn't work